.github/workflows/stale.yml

```diff
@@ -36,3 +36,7 @@ jobs:
         stale-pr-label: 'no-pr-activity'
         exempt-pr-labels: 'awaiting-approval,work-in-progress'
         only-labels: 'awaiting-feedback,awaiting-answers'
+        days-before-pr-stale: '10'
+        days-before-pr-close: '20'
```

Comment:
I have PRs that are probably more than 20 days that I don't want be closed automatically (those with `RAW` in their title). Will I have to re-open them? Maybe we should only label the PRs as stale, but not automatically close them (since there aren't a lot of PRs anyway, it should not add to much work for us). wdyt?

Do PRs labeled "Draft" also get closed automatically?

Reply:
We can add a label that will not mark the PR as stale or close it. For example, this label will avoid it as stale`work-in-progress`

Reply:
sgtm. Let's pilot it and see how well it works then :-)